.github/dependabot.yml

```
@@ -20,6 +20,10 @@ updates:
     schedule:
       interval: "weekly"
       day: "friday"
+    ignore:
+      # post-1.3 versions require Java 11
+      - dependency-name: "ch.qos.logback:logback-*"
+        update-types: [ "version-update:semver-major", "version-update:semver-minor" ]
```

```suggestion
        versions: [ "[1.4,)" ]
```
Maybe it is better to provide an explicit version range to exclude? I have already seen Dependabot confused regarding what constitutes a minor version bump.

---

I would prefer that - are you sure dependabot supports that syntax? https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#versions-ignore doesn't mention Maven, so it *should*, but you know how it is with assumptions :)